regulations/static/regulations/css/scss/module/comment-review.scss

```diff
@@ -185,19 +185,19 @@ comment-review.scss contains styles for Review Your Comment page and the PDF it
     a.btn-group-item {
       color: $black;
       display: inline-block;
-      background: $grey;
+      background: $gray_lighter;
       font-size: 14px;
       line-height: 18px;
       padding: 8px 15px;
-      box-shadow: inset 0 0 1px fade($dark_gray, 90%);
+      box-shadow: inset 0 0 1px fade($gray_light, 90%);
       border-radius: 2px;
       cursor: pointer;
 
       &:hover {
-        background: darken($grey, 10%);
+        background: darken($gray_lighter, 10%);
```

Comment:
I knew I was forgetting something! I replaced these with the next darkest color (so `darken($gray_lighter, 10%);` just becomes `$gray_light`) and I think it works in most cases though our `$gray_light` and `$gray_lighter` may be too close together and `$gray_light` may need to be a bit darker across the entire app.